src/modules/polls.ts

```diff
@@ -89,8 +88,8 @@ export const getNewEmbed = (
 	return oldEmbed;
 };
 
-export const sendAttendanceEmbed = async (
-	embed: AttendancePoll,
+export const sendPollEmbed = async (
+	embed: Poll,
```

Comment:
(Not relative to this PR, but since we're touching this), this parameter should probably have a better name

Reply:
Could just call it poll, would make sense